### DISCUSSION TITLE
Target contract cannot be found

### DISCUSSION THREAD

**Discussion by prakharmishra3956** [Cyfrin/foundry-full-course-cu#2629] [Q&A] [answered]
![Screenshot (35)](https://github.com/user-attachments/assets/77a5e278-6cec-414d-be07-bc6a1ec7663d)

**Comment by EngrPips** [comment, marked as answer]
The command is supposed to be `forge script script/DeploySimpleStorage.s.sol:DeploySimpleStorage`

**Reply by prakharmishra3956** [reply]
Now there's another error -
![Screenshot (36)](https://github.com/user-attachments/assets/84c54ca6-bdb5-4992-a5f0-a2eedca82035)

**Reply by EngrPips** [reply]
Please delete the `cache` folder, then run `forge clean` then retry the process.

**Comment by prakharmishra3956** [comment]
Thanks, it's resolved.

**Reply by EngrPips** [reply]
Awesome, Keep crushing it.
